src/App.js

```diff
@@ -4,9 +4,24 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  savedQuery = (event) => {
```

Comment:
```suggestion
  saveQuery = (event) => {
```
Name of the function should describe what it actually does